### ISSUE TITLE
-Wimplicit-fallthrough in drivers/net/ethernet/asix/ax88796c_main.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1491]
```
drivers/net/ethernet/asix/ax88796c_main.c:696:2: error: unannotated fall-through between switch labels [-Werror,-Wimplicit-fallthrough]
        case SPEED_10:
        ^
drivers/net/ethernet/asix/ax88796c_main.c:696:2: note: insert 'break;' to avoid fall-through
        case SPEED_10:
        ^
        break;
drivers/net/ethernet/asix/ax88796c_main.c:706:2: error: unannotated fall-through between switch labels [-Werror,-Wimplicit-fallthrough]
        case DUPLEX_HALF:
        ^
drivers/net/ethernet/asix/ax88796c_main.c:706:2: note: insert 'break;' to avoid fall-through
        case DUPLEX_HALF:
        ^
        break;
```

Patch submitted: https://lore.kernel.org/r/20211025211238.178768-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/netdev/net-next/c/3c5548812a0c

**Comment by nathanchance** [comment]
Fixed in next-20211027.
